src/views/UserEdit/UserEdit.js

```diff
@@ -154,6 +154,12 @@ class UserEdit extends React.Component {
       .catch((e) => showErrorCallout(e, this.context.sendCallout));
   }
 
+  externalSystemIdValidator = (user) => {
+    if (!user.externalSystemId?.trim()) {
+      delete user.externalSystemId;
+    }
+  };
+
```

Comment:
I think this function can be generalized to work for any property within user. example - lines 171 - 173

Reply:
Agree with @Terala-Priyanka, this can be a general function like `deleteEmptyUserField(field)`